packages/fake-driver/package.json

```diff
@@ -23,14 +23,10 @@
   },
   "license": "Apache-2.0",
   "author": "https://github.com/appium",
-  "bin": {
-    "appium-fake-driver": "index.js"
-  },
```

Comment:
This entrypoint is also listed in `package-lock.json`, should that be updated as well?
Also I guess this is technically a breaking change, but it was never documented (and the driver also isn't meant to be used outside tests in the first place), so it might be okay.

Reply:
yes, fake driver is only used for internal testing, so it is fine